packages/vuepress-theme-odyssey/styles/index.scss

```diff
@@ -104,7 +104,7 @@ h4,
 h5,
 h6 {
   .header-anchor {
-    display: inline-block;
+    display: none;
```

Comment:
Is this going to be set back in the future?

Reply:
@arnoldsandoval-okta, i think this was your call - context?

Reply:
Yes! Currently these dont work because we have content in hidden divs. Its weird, they will come back though!

Reply:
Should we just remove this from this release then?